README.md

```diff
@@ -1,17 +1,64 @@
-# Multi selection list
+# Multi select
 
-Kenshoo multi selection list component.
+[![Build Status](https://travis-ci.org/kenshoo/react-multi-select.svg?branch=master)](https://travis-ci.org/kenshoo/react-multi-select) [![GitHub version](https://badge.fury.io/gh/kenshoo%2Freact-multi-select.svg)](https://badge.fury.io/gh/kenshoo%2Freact-multi-select)
 
+**Kenshoo multi select component**
 
-<!-- example -->
+Multi select is a straight forward component that helps a user select multiple items in a clear and filterable way.
+
+![Preview](preview.png?raw=true "Preview")
+
+## Installation
+ 
+ **Installation using npm:**
+ 
+ ```
+ npm install @kenhooui/react-multi-select --save
```

Comment:
is --save is unnecessary?

Reply:
without it it would not be added to the package.json